### DISCUSSION TITLE
Setting for turning off notifications about downloaded beatmap

### DISCUSSION THREAD

**Discussion by gotarik** [ppy/osu#27612] [Ideas] [answered]
When I download many maps from the beatmap list, I get a lot of notifications. Because of this lags begin. It would be good if there was a button in the settings that disables notifications about each downloaded beatmap

**Comment by peppy** [comment]
Notifications should not cause lag. Could you please provide a video of this?

**Reply by gotarik** [reply]
[with notifications1](https://youtu.be/33rtjzkDlm8) [with notifications2](https://youtu.be/r5G05o-H2Bw) [without notifications1](https://youtu.be/BFLAIOjL9jY) [without notifications2](https://youtu.be/MH7dBkTtWFQ) it lags without notifications, when i choose beatmap for the first time. I suppose it's because osu is on SSD and beatmaps on HDD, but it lags only for the first time. But with notifications it lags many times in a row.

**Reply by peppy** [reply]
What are you doing to cause that many beatmap import notifcations???

**Reply by gotarik** [reply]
Uhmm... download beatmaps?....

**Reply by peppy** [reply]
Please provide a screen capture showing what you're doing

**Reply by gotarik** [reply]
https://www.youtube.com/watch?v=G58S14y91zM

**Reply by peppy** [reply, marked as answer]
Okay, so Ithas nothign to do with notifications. It's hte import process. You're just making weird assumptions.

There are some overhaeds with realm and large databases, it will continue to be optimised.
